core/src/main/java/tech/tablesaw/api/BooleanColumn.java

```diff
@@ -495,6 +495,18 @@ public BooleanColumn set(Selection rowSelection, boolean newValue) {
     return this;
   }
 
+  @Override
+  public Column<Boolean> setObj(int row, Object obj) {
```

Comment:
I don't see `setObj` called at all in this PR. Can we drop it from the PR? If it's necessary for you let's consider in a different PR to keep this one from getting too large